migrations/v7.js

```diff
@@ -0,0 +1,109 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-accordion - v5.3.0 > v7.0.0', async () => {
```

Comment:
`_imageAlignment` came in `v7.3.0` rather than `v7.0.0`

Reply:
Thank you.

Reply:
Minor thing, but for consistency, consider tweaking the version references to reflect that the change was from `7.2.3` to `7.3.0`